Documentation/policy/language.rst

```diff
@@ -143,8 +143,9 @@ the label ``role=frontend``.
 Egress Allow All
 ~~~~~~~~~~~~~~~~~
 
-An empty `EndpointSelector` will select all endpoints, thus writing a rule that will allow
-all egress traffic from an endpoint may be done as follows:
+An empty `EndpointSelector` will select all endpoints in the pod's
```

Comment:
Reads a bit awkward. How about this - "will select all egress endpoints from an endpoint based on the CNP namespace ("default" ...)?
```suggestion
An empty `EndpointSelector` will select all endpoints in the pod's
```

Reply:
I think the first clause in that sentence is not specific to the following example. That is, `EndpointSelector` isn't specific to egress :-)

We could break into two sentences if the current form sounds weird.

Reply:
Since it's listed under the `Egress` section, further emphasized by the second clause, it doesn't confuse me. So I'll leave the decision to @twpayne.

Reply:
OK, I've split this into two sentences which hopefully makes it clearer.